src/components/ButtonPanel.js

```diff
@@ -1,41 +1,46 @@
 import React from 'react'
 import Button from './Button'
 
-const ButtonPanel = (props) => {
-
-    return (
-        <div className="button-panel">
-            <div className="group-1">
-                <Button name="AC" />
-                <Button name="+/-" />
-                <Button name="%" />
-                <Button color="orange" name="/" />
-            </div>
-            <div className="group-2">
-                <Button name="7" />
-                <Button name="8" />
-                <Button name="9" />
-                <Button color="orange" name="X" />
-            </div>
-            <div className="group-3">
-                <Button name="4" />
-                <Button name="5" />
-                <Button name="6" />
-                <Button color="orange" name="-" />
-            </div>
-            <div className="group-4">
-                <Button name="1" />
-                <Button name="2" />
-                <Button name="3" />
-                <Button color="orange" name="+" />
-            </div>
-            <div className="group-5">
-                <Button name="0" special={true}/>
-                <Button name="." />
-                <Button color="orange" name="=" />
+class ButtonPanel extends React.Component {
+    handleClick = (btnName) => {
```

Comment:
As you will observe, we are repetitively declaring `handleClick` here in `ButtonPanel` and once again in `Button`. 

`ButtonPanel` should simply pass `clickHandler` as a prop to `Button`. It is not reponsible for actually calling on the `clickHandler` method it received from `App`.

As such we don't need `handleClick` here in `ButtonPanel`. Let us simply pass `clickHandler` into `Button` component.